src/server/assets/index.js

```diff
@@ -33,4 +33,15 @@ const getAssetsArray = () => {
   return assets;
 };
 
-export default getAssetsArray;
+const getAssetDomains = () => {
+  const assetDomains = ['ichef.bbci.co.uk', 'gel.files.bbci.co.uk'];
+
+  const simorghStaticAssetsHost = process.env.SIMORGH_PUBLIC_STATIC_ASSETS_HOST;
```

Comment:
When this isn't set in the local config does that mean we won't prefetch or preconnect any domains?

Reply:
It would just avoid preconnecting localhost, but it's probably overcomplicating things.